spyder/config/main.py

```diff
@@ -367,6 +367,8 @@
               'editor/delete line': 'Ctrl+D',
               'editor/transform to uppercase': 'Ctrl+Shift+U',
               'editor/transform to lowercase': 'Ctrl+U',
+              'editor/indent': 'Ctrl+[',
+              'editor/unindent': 'Ctrl+]',

```

Comment:
@rlaverde, these shortcuts are used in the reversed, i.e.

* `Ctrl + ]` -> Indent
* `Ctrl + [` -> Unindent

At least that's the case in Sublime: https://gist.github.com/eteanga/1736542

So please open a new PR to fix that.